docs/Language Feature Status.md

```diff
@@ -6,7 +6,7 @@ and will be updated as work progresses, features are added / removed, and as wor
 This is not an exhaustive list of our features but rather the ones which have active development
 efforts behind them.
 
-# C# 7.1
+# C# 7.1 (Roslyn 2.3.x)
```

Comment:
Are you aware of this page for documenting packages? https://github.com/dotnet/roslyn/wiki/NuGet-packages (there is a link from our recent nuget packages to that page)
I think it would be better not to duplicate the information (harder to maintain).

Reply:
I was not aware of that page, and it's pretty much exactly what I was looking for. I'll close this PR, thank you very much!